rclpy/src/rclpy/_rclpy.c

```diff
@@ -834,7 +830,7 @@ rclpy_destroy_node_entity(PyObject * Py_UNUSED(self), PyObject * args)
   }
 
   rcl_node_t * node = (rcl_node_t *)PyCapsule_GetPointer(pynode, NULL);
-  rcl_ret_t ret;
+  rcl_ret_t ret = RCL_RET_ERROR;
```

Comment:
It looks like the current linter doesn't like the even unchanged code anymore. I initialized the value in the `else` block in c15271beb25344bab31edca1ad97fcb9c3118217